### PR TITLE
Match most of 9FC031D0.c

### PR DESCRIPTION
**Pull request by pixel-stuck**
3 functions left.

### CODE REVIEW

File: include/bbtypes.h
Comment (Thar0) [inline include/bbtypes.h]:
```suggestion
#endif

```

File: src/9FC031D0.c
Comment (Thar0) [inline src/9FC031D0.c]:
```suggestion
    for(i = 0; i < ARRAY_COUNT(virage2_offset->publicKey); i++) {
```
And same for the `wordcopy` calls below?

File: src/9FC031D0.c
Comment (Thar0) [inline src/9FC031D0.c]:
```suggestion
    IO_WRITE(PI_ALLOWED_IO, cmdHead->hwAccessRights & 0xFF);
```
Since the others are expressed in terms of masks and shifts, this could be more uniform

**Review by Thar0 — COMMENTED** [review]
Couple of very small things

File: src/9FC031D0.c
Comment (pixel-stuck) [reply src/9FC031D0.c]: Happy to do this, though I think ARRAY_COUNT isn't in yet (added in your currently open PR), so I didn't want to step on any toes and I figured we could clean it up later. I can just wait until yours is merged to address this.

**Review by Thar0 — APPROVED** [review]

**Pull request merged**
